packages/backend/src/routes/webhooks.ts

```diff
@@ -13,9 +13,24 @@ import webhookHandler from '@/controllers/webhooks/handler'
 import logger from '@/helpers/logger'
 
 const router = Router()
-const upload = multer()
+const uploadNone = multer().none()
 
-router.use(upload.none())
+router.use((req, res, next) => {
+  uploadNone(req, res, (err) => {
+    // file upload is not supported
+    // handle error to prevent http 500 error
+    logger.error({
+      req,
+      err,
+      msg: 'Invalid request: multer error',
+    })
+    if (err instanceof multer.MulterError) {
+      res.status(400).send('Invalid request, file upload is not supported.')
```

Comment:
nit: consider using a more specific HTTP error code here
```suggestion
      // Unprocessable entity
      res.status(422).send('Invalid request, file upload is not supported.')
```